src/bl/attachmentFacade.ts

```diff
@@ -1,32 +1,33 @@
 import { getManager } from 'typeorm'
 
 import { NotFoundException } from './errors/NotFoundException'
-import { Attachement } from '../entities/attachement'
 import { ParamsExtractor } from './paramsExtractor'
 import { CardFacade } from './cardFacade'
-export class AttachementFacade {
+import { Attachment } from './../entities/attachment'
 
-    static async getAllFromCardId(cardId: number): Promise<Attachement[]> {
+export class AttachmentFacade {
+
+    static async delete(attachmentId: number): Promise<boolean> {
         try {
-            const card = await CardFacade.getById(cardId)
-            const attachements = card.attachements
-            if (attachements) {
-                return attachements
+            const deletionSuccess = await getManager()
+                    .getRepository(Attachment)
+                    .removeById(attachmentId)
```

Comment:
removeById return nothing, so the rest of the function is useless. Please see ListFaced.